common/auth/tlsConfigHelper.go

```diff
@@ -0,0 +1,59 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
you may want to remove this line?

Reply:
Actually, no. We discussed this awhile ago and decided to keep header the same on all files. Because code can be moved between files and it is easy to miss if copyrights are still valid.